test/runtime/src/lib.rs

```diff
@@ -286,6 +286,7 @@ impl cumulus_pallet_parachain_system::Config for Runtime {
 	type XcmpMessageHandler = ();
 	type ReservedXcmpWeight = ();
 	type CheckAssociatedRelayNumber = cumulus_pallet_parachain_system::AnyRelayNumber;
+	#[cfg(feature = "parameterized-consensus-hook")]
```

Comment:
likewise here - it seems fine for the test runtime (which is effectively a binary crate used internally, not a library) to explicitly use the `RequireParentIncluded`

Reply:
Which way is best to get this to compile without the feature flag here? 

![Screenshot 2023-07-19 at 5 02 57 PM](https://github.com/paritytech/cumulus/assets/34992650/c815b456-187d-45cd-8b33-ebc722a4b2ac)

I expect some change is needed around line 215 of `pallets/parachain-system/src/lib.rs` such that it can still accept a `ConsensusHook` type even without the feature flag enabled. Just haven't seen an example of what that would look like.

Reply:
Can we just have it unconditionally enable the crate feature in its `Cargo.toml`?

Reply:
That does make sense 👍 Will be back on to try it this evening.

Reply:
Robert probably meant doing
```toml
cumulus-pallet-parachain-system = { path = "../../../../pallets/parachain-system", default-features = false, features = ["parameterized-consensus-hook"] }
```
instead of introducing a feature to test-crates.